switch to PyVisa, new LOOP module

SamSchott opened the pull request:
- moved communication with instruments to interface independent PyVisa with pyvisa-py as default backend (no installation of separate visa drivers necessary)
- singleton behavior: always return the same instance MercuryITC instance for an existing connection / address
- added LOOP module to hold all control loop settings (PID values, etc)

crazyfermions commented:
Hi Sam, thanks for the work you put into this. I am leaning towards basically merging everything, however I would like to maintain the changelog you deleted 
(https://github.com/crazyfermions/python-mercury_driver/pull/1/commits/b9f9935c83910dd85215cfa10023908eee382cb1) and I would also like to maintain the link to this repository in the setup.py and also keep my name and contact data in the authors list -- feel free to add your contact data there as well.

SamSchott commented:
Of course, I have just added the changelog again and reverted the setup.py information. And thank you for a great driver -- we are using it almost daily in our lab.
